carla/__init__.py

```diff
@@ -1,5 +1,7 @@
 # flake8: noqa
 
+from _version import __version__
```

Comment:
@Philoso-Fish That doesn't seem to work as intended. The tests fail: https://github.com/indyfree/CARLA/pull/95/checks?check_run_id=3184224392#step:4:309

Reply:
Looks like I forgot a dot :smile: 
Will change that!

Reply:
Looks like I forgot a dot :smile: 
Will change that!